ccd-definition/AuthorisationCaseEvent.json

```diff
@@ -34,13 +34,27 @@
     "UserRole": "caseworker-publiclaw-solicitor",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "amendChildren",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "CRU"
+  },
```

Comment:
I think I would prefer these events bunched together by role. Maybe not important though, I will eventually be splitting these out into separate json files

Reply:
Makes sense - I grouped them by role.